.github/workflows/python_tests.yml

```diff
@@ -40,7 +40,10 @@ jobs:
           uv run pytest -n ${{ inputs.parallelism }} --durations=0 -v -s --log-cli-level=DEBUG --no-skip-cached-tests ${{ inputs.pytest-add-params }}
 
       - name: Upload coverage
-        if: inputs.coverage-flags != ''
+        if:
```

Comment:
what is this for?

Reply:
nothing related, I just migrated to another Mac and committed everything that was wip